modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -49,6 +57,11 @@
     protected int serverPort = 8080;
     protected String serverHost = "0.0.0.0";
     protected String aspnetCoreVersion= "2.1"; // default to 2.1
+    private String classModifier = "";
+    private String operationModifier = "virtual";
```

Comment:
Is this intended to always be the default? Looking at the logic, it seems a user can specify a protected class modifier and empty operation modifier.

Reply:
An operation need not have a modifier so I think it is OK. I suspect we can create odd (and probably illegal) combinations such as a real class with an abstract method but that is a significantly bigger exercise to check for and possibly not worth it in the generator.